src/sentry/static/sentry/app/components/autoComplete.jsx

```diff
@@ -185,7 +185,7 @@ class AutoComplete extends React.Component {
       onOpen(...args);
       return;
     }
-
+    this.resetHighlightState();
```

Comment:
highlight state is weird when it persists across open/close imo

Reply:
Agreed, thought this was weird

Reply:
Can you add a test for this?